pages/roles-and-teams.md

```diff
@@ -2,60 +2,26 @@
 title: Roles and teams
 permalink: /roles-and-teams/
 ---
-Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles on a number of different teams across 18F. Below are a few of the roles that Innovation Specialists may fill at 18F. This is not intended to be a comprehensive list but for those that are interested in joining the team, hopefully something on this list excites you!
+Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles - from software engineers and product leads, to UX, visual designers and content strategists on a number of different teams across 18F. Our needs change frequently, so check back often!
 
-### Comms Team
+**Current needs**
 
-**Director of Outreach**
-
-The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
+**Business Strategists - Digital Transformation**
 
-### Consulting Team
+Business strategists help the agencies deliver on their mission through insights that guide strategy and solve problems for end users. They use workshops and human centered design principles as a means to collaborating with partner agencies, and getting results. 
 
-**Acquisition Specialist**
+**Director of Outreach**
 
-Acquisition specialists help teams support digital services through procurement efforts, investigate other agencies’ tools and processes to find opportunities to improve results, and work with contracting officers to help procure goods and services for 18F and 18F partner agencies.
+The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
```

Comment:
Suggested change: 'messaging, and general community evangelism' to 'and messaging, as well as general community evangelism.'